src/App.js

```diff
@@ -2,21 +2,40 @@ import "./App.css";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 import { useState } from "react";
+import { useEffect } from "react";
+
+const useSemiPersistentState =()=>{
```

Comment:
great job on the hook
but think about what happens when your app is loaded for first time and it is nothing saved in localStorage?
what is the value of todoList in this case? what tipe or error you will receive and how you can solve it?